schemas/livePaperVersion.schema.tpl.json

```diff
@@ -0,0 +1,59 @@
+{
+  "_type": "https://openminds.ebrains.eu/publications/LivePaperVersion",
+  "_extends": "/core/products/researchProductVersion.schema.tpl.json",
+  "required": ["digitalIdentifier", "license"],
+  "properties": {
+    "about": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add links to the datasets, models or software contained within or described by this version of the live paper.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/DatasetVersion",
+        "https://openminds.ebrains.eu/core/ModelVersion",
+        "https://openminds.ebrains.eu/core/SoftwareVersion"
+      ]
+    },
+    "abstract": {
+      "type": "string",
+      "_instruction": "Enter the abstract of the live paper."
+    },
+    "author": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "If necessary, add one or several authors (person or organization) that contributed to the production and publication of this live paper version. Note that these authors will overwrite the ones provided in the live paper product this version belongs to.",
+      "_linkedCategories": ["legalPerson"]
+    },
+    "creationDate": {
```

Comment:
what is to be captured here? for all other research products we decided in the past that only the date where the item is publicly released is relevant. If we need to have creation date captured we should do this for all ResearchProductVersions

Reply:
for research products that are developed over time through an authoring UI before being released, like live papers, we really need a date field that can be used pre-publication, for sorting.

Thinking about this, though, "lastModifiedDate" would be more useful. I will make that change here, and then if you think this makes sense for other ResearchProducts we can move it to core in a future PR.